cypress/integration/views/logsView.ts

```diff
@@ -25,6 +25,20 @@ describe('Logs page', () => {
         cy.dataName('logOccurenceDurationSelectedOption').should('contain.text', '15 Minutes')
       })
     })
-  })
 
+    context('when starting a flow',  () => {
+      it('should display the correct logging', () => {
+        cy.visit('#/flow')
+        cy.dataName('playButton').click()
+        cy.visit('#/logs')
+        cy.wait(5000)
```

Comment:
`cy.get` has a timeout argument https://docs.cypress.io/api/commands/get.html#Timeouts

Rather than waiting 5 seconds which could be flaky because different machines might take different time, how about using `cy.get` with a higher timeout?

Reply:
That is a very good idea!

Reply:
done